projects/cli-files/go-cat/cmd/root.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+)
+
+func Execute() {
+	path, err := checkArg()
+	if err != nil {
+		fmt.Println(err)
+	} else {
+		data, err := os.ReadFile(path)
```

Comment:
This will read the whole file into memory. This may cause problems e.g. if your computer with 16GB RAM tries to read a 100GB file.

Can you think of a way to achieve the same behaviour, but to avoid needing to read the whole file into memory?